src/main/java/org/opentripplanner/middleware/utils/ItineraryUtils.java

```diff
@@ -110,22 +110,22 @@ public static boolean itineraryIsSavable(Itinerary itinerary) {
     /**
      * Returns true if the itineraries match for the purposes of trip monitoring.
      *
-     * @param previousItinerary The original itinerary that others are compared against.
-     * @param newItinerary A new itinerary that might match the previous itinerary.
+     * @param referenceItinerary The original itinerary that others are compared against.
+     * @param candidiateItinerary A new itinerary that might match the previous itinerary.
```

Comment:
Oops, minor typo: candidiateItinerary -> candidateItinerary

Reply:
Fixed in 02831be08c58fc0a47a12eb5f6927eddd76593df